docs/reference/project_template.md

```diff
@@ -1,6 +1,8 @@
+(project_template)=
```

Comment:
CI is generating this error. 

> generating indices... /home/docs/checkouts/readthedocs.org/user_builds/wagtail/checkouts/11973/docs/reference/management_commands.md:15: WARNING: more than one target found for 'myst' cross-reference project_template: could be :std:ref:`The project template` or :std:doc:`The project template` [myst.xref_ambiguous]


You will either need to change this ref to something that's not the page name.

```suggestion
(reference_project_template)=
```

Or, change your links to reference the page, not the ref.